src/rp2040/boot_stage2/include/boot_stage2/config.h

```diff
@@ -11,6 +11,8 @@
 
 #include "pico.h"
 
+// PICO_CONFIG: PICO_FLASH_SPI_CLKDIV, Clock divider from clk_sys to use for serial flash communications in boot stage 2. On RP2040 this must be a multiple of 2, type=int, default=varies; often specified in board header, advanced=true, group=boot_stage2
```

Comment:
Just spotted https://forums.raspberrypi.com/viewtopic.php?t=390296 - maybe worth mentioning that you have to set this on the `bs2_default` target (or as `INTERFACE` on `boot_stage2_headers`) not your regular target when setting from CMakeLists.txt? Or even mention that in the description for `boot_stage2` if there is one, as this applies to all the boot_stage2 defines?